### DISCUSSION TITLE
Issue: in threads just the first tweet it's visible (so not a thread)

### DISCUSSION THREAD

**Discussion by Nehemoth** [robertodoering/harpy#721] [Ideas]
For some reason when you want to see a twitter thread just the first tweet appears. 
When you open a thread you expect to see all the tweets together once after the other in a cohesive way to read the whole thread, in this case you only see the first tweet so you can't see the thread at all.

**Comment by Ysasmendi** [comment]
I would add to the above that it is impossible to find the rest of the thread even when going to the author's timeline, which makes Harpy unusable until this is fixed. Thanks.
